src/Storage/Commands.Storage/Blob/Cmdlet/StopAzureStorageBlobCopy.cs

```diff
@@ -89,6 +91,12 @@ public string CopyId
         }
         private string copyId;
 
+        protected override void BeginProcessing()
+        {
+            base.BeginProcessing();
+            OutputStream.ConfirmWriter = (s1, s2, s3) => ShouldContinue(s2, s3);
+        }
+
```

Comment:
In your code, you prompt for confirmation only when there is going to be an overwrite - this ensures that, in these cases, you will use ShouldContinue, which always prompts, unless the user specifies -Force